src/Engine/Data/VolumetricMaterial.hpp

```diff
@@ -94,6 +94,7 @@ class RA_ENGINE_API VolumetricMaterial final : public Material, public Parameter
 
   private:
     // The density matrix
+    /// \todo get rid of the raw ptr.
```

Comment:
Should we use same logic than for other materials. i.e. inheriting from `public MaterialTextureSet<TextureSemantics::VolumetricMaterial>` ?

Reply:
If I understand well, Volumetric Material has only one texture and manage it.
So in it's current form, using TextureSet is overkill.

Reply:
Right, this was due to the old way textures were managed and loaded. Here, the 3D texture is loaded as part of the Asset::VolumeData and converted to VolumetricMaterial by the VolumeComponent. But I think that unifying material interface should help even if this specific part can be delayed for a future PR.

Reply:
Let's delay this for a futur PR.